pages/learn.js

```diff
@@ -22,6 +22,9 @@ export default () => {
           </p>
         </div>
       ))}
+      <div>
+        <p>Feel free to add suggestios to learning resources <a href='https://github.com/copenhagenjs/copenhagenjs.dk/blob/master/data/learn.js'>here</a></p>
```

Comment:
"here" is not a good link text for accessibility.

"Do you know a good resource to learn JavaScript? You should <a href="..">Submit a Pull Request here</a>."

Reply:
Thanks, Kevin. I'll change it to that.